### PR TITLE
Initial Draft: Added Institution Collection Camp Contribution

### PR DESCRIPTION
**Pull request by belwalshubham**
image

## Summary by CodeRabbit

- **New Features**
	- Introduced support for a new target type: 'institution-collection-camp'.
	- Added links for material contributions specific to the institution collection camp.
	- Enhanced rendering logic to display relevant information for the new target type.

- **Bug Fixes**
	- Maintained existing error handling for date creation and data retrieval processes.

### CODE REVIEW

File: wp-content/plugins/goonj-blocks/goonj-blocks.php [hunk 1]
```diff
@@ -86,8 +86,11 @@ function gb_goonj_blocks_check_action_target_exists() {
 		'Dropping_Centre.State',
 		'Dropping_Centre.District_City',
 		'Collection_Camp_Core_Details.Contact_Id.display_name',
+		'Institution_Collection_Camp_Intent.Collections_will_start_on_Date',
+		'Institution_Collection_Camp_Intent.Collections_will_end_on_Date_',
+		'Institution_Collection_Camp_Intent.Collection_Camp_Address'
 	);
-
+
```
Comment (tarunnjoshi) [inline wp-content/plugins/goonj-blocks/goonj-blocks.php:93]: Why these extra spaces here?

**Review by tarunnjoshi — CHANGES_REQUESTED** [review]

File: wp-content/plugins/goonj-blocks/build/render.php [hunk 2]
```diff
@@ -14,10 +14,10 @@
 
 $slots = generate_induction_slots($source_contact_id);
 
-
 $headings = [
   'collection-camp' => 'Collection Camp',
   'dropping-center' => 'Dropping Center',
+  'institution-collection-camp' => 'Collection Camp',
```
[the remark this answers is not in the document]
Comment (belwalshubham) [reply wp-content/plugins/goonj-blocks/build/render.php:20]: For now, let's proceed with using "collection camp," but I'll confirm with the Goonj team on Monday. I'll make a note of it, and we can change it later if needed.

File: wp-content/plugins/goonj-blocks/goonj-blocks.php (hunk 1 above)
Comment (belwalshubham) [reply wp-content/plugins/goonj-blocks/goonj-blocks.php]: fixed

File: wp-content/plugins/goonj-blocks/build/render.php (hunk 2 above)
[the remark this answers is not in the document]
Comment (tarunnjoshi) [reply wp-content/plugins/goonj-blocks/build/render.php:20]: Sure please do. Let's merge this for now

**Review by tarunnjoshi — APPROVED** [review]

**Pull request merged**